pandas/core/resample.py

```diff
@@ -503,6 +503,53 @@ def ffill(self, limit: int | None = None):
         --------
         Series.fillna: Fill NA/NaN values using the specified method.
         DataFrame.fillna: Fill NA/NaN values using the specified method.
+
+        Examples
+        --------
+        Here we only create a ``Series``.
+
+        >>> ser = pd.Series([1, 2, 3, 4], index=pd.DatetimeIndex(
+        ...                 ['2023-01-01', '2023-01-15', '2023-02-01', '2023-02-15']))
+        >>> ser
+        2023-01-01    1
+        2023-01-15    2
+        2023-02-01    3
+        2023-02-15    4
+        dtype: int64
+
+        Example for ``ffill`` with downsampling (we have fewer dates after resampling):
+
+        >>> ser.resample('M').ffill()
+        2023-01-31    2
+        2023-02-28    4
+        Freq: M, dtype: int64
+
+        Example for ``ffill`` with upsampling (we have more dates after resampling):
```

Comment:
```suggestion
        Example for ``ffill`` with upsampling (fill the new dates with the previous value):
```

Might be a little more clear

Reply:
Yes, thank you.